### ISSUE TITLE
What if you pull a bite you have already solved?

### ISSUE THREAD

**Issue by bbelderbos** [PyBites-Open-Source/eatlocal#61]
I made a demo video today for our v2 series and it puzzled me a bit that I got a solution pulled in instead of a template file. This was because I resolved it on the platform (and forgot). Is this just user error or should we show a message then and/or add a reset switch?

**Comment by rhelmstedter** [comment]
All `eatlocal` does is pull down whatever is already on the platform, including solutions.

In v1, the platform would not let you refactor after submission unless you complete reset the bite (remember there was the _always be coding_ gif) With v2, you don't have to reset the bite to refactor and run the tests. 

That being said, I can see the value in having the ability to _reset_ if the user would like to solve the bite again instead of solely refactoring.

**Comment by bbelderbos** [comment]
Yeah it would be nice if there is a reset flag, but definitely low prio, let's have more people request it first.

**Comment by bbelderbos** [comment]
I almost want to close this, people can reset the Bite on the website - is this even within the scope of Eatlocal?

**Comment by bbelderbos** [comment]
If anything this is more urgent: https://github.com/PyBites-Open-Source/eatlocal/issues/57

**Comment by Pradhvan** [comment]
> I almost want to close this, people can reset the Bite on the website - is this even within the scope of Eatlocal?

@bbelderbos is this within the scope of eatlocal ? 

If yes and you think I can pick it up (assuming I only need to know about eatlocal to fix the problem), if @rhelmstedter isn't already working on it ?

**Comment by bbelderbos** [comment]
This might require some platform knowledge but happy if you want to take a look and think about a solution. Thanks.

**Comment by Pradhvan** [comment]
> This might require some platform knowledge but happy if you want to take a look and think about a solution. Thanks.

sure I can brain storm and start a discussion going around in this thread.

**Comment by bbelderbos** [comment]
> > This might require some platform knowledge but happy if you want to take a look and think about a solution. Thanks.
> 
> sure I can brain storm and start a discussion going around in this thread.

Sorry this comment was in relation to https://github.com/PyBites-Open-Source/eatlocal/issues/57 - this issue is mostly eatlocal scoped, thanks @rhelmstedter for pointing to the right button to click - last option is "reset" indeed.

**Comment by Pradhvan** [comment]
> Sorry this comment was in relation to https://github.com/PyBites-Open-Source/eatlocal/issues/57 - this issue is mostly eatlocal scoped, thanks @rhelmstedter for pointing to the right button to click - last option is "reset" indeed.

No problem Bob


> I've never selected from a drop down with playwright, but I have done it with selenium so I imagine its doable. Hopefully it will be as simple as finding the button, choosing reset, and then downloading as normal. I circled the button in the screenshot below.
> 
> ![Image](https://github.com/user-attachments/assets/62bbdf2f-a2d5-4d19-bd18-b55ae9241fe3)

This makes more sense now, I have done something similar with Selenium too. Lemme see how that can be done with playwright. Hopefully that should do it. Thanks @rhelmstedter
